nd/firefox/browser/chrome/browser/aboutSessionRestore.dtd.po

```diff
@@ -123,6 +123,9 @@ msgid ""
 "Your add-ons and customizations have been removed and your browser settings "
 "have been restored to their defaults. If this didn't fix your issue, "
 msgstr ""
+"Ama add-on akho lokulungisa kwakho sokukhutshiwe kwathi amasethingi e-"
+"browser yakho abiselwa esimweni kuqala. Nxa lokhu kungalungisanga inking "
```

Comment:
`Ama add-on akho lokulungisa kwakho sokukhutshiwe kwathi amasethingi e-"
"browser yakho abiselwa esimweni kuqala` should read

Ama add-on akho lokulungisa kwakho sokukhutshiwe kwathi amasethingi e-"
"browser yakho abiselwa **esimweni sakuqala**